<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisAssayData</class>
 <widget class="QDialog" name="DisAssayData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_DevIndex">
       <property name="text">
        <string>仪器编号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_DevIndex"/>
     </item>
     <item>
      <widget class="QLabel" name="label_StartEndTime">
       <property name="text">
        <string>起止时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_Start">
       <property name="displayFormat">
        <string>yyyy年M月d日</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_End">
       <property name="displayFormat">
        <string>yyyy年M月d日</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_PipeIndex">
       <property name="text">
        <string>管道编号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PipeIndexStart"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PipeIndexEnd"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Query">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
